tdrs-backend/tdpservice/reports/test/test_api.py

```diff
@@ -119,3 +119,16 @@ def test_reports_data_prepper_not_allowed(api_client, data_prepper):
 
     response = api_client.post("/v1/reports/", data)
     assert response.status_code == status.HTTP_403_FORBIDDEN
+
+@pytest.mark.django_db
+def test_s3_signed_url(api_client, user):
+    """Test that a url string is given to the client."""
+    api_client.login(username=user.username, password="test_password")
+    response = api_client.post("/v1/reports/signed_url/", {
+        "file_name": "test.txt",
+        "file_type": "plain/text",
+        "client_method": "put_object"
+    })
+
+    assert response.status_code == status.HTTP_200_OK
+    assert response.data['signed_url']
```

Comment:
It may make sense to split this into two tests - one that tests the `get_object` client_method and another for `put_object`.

Using [Pytest Parametrization](https://docs.pytest.org/en/stable/example/parametrize.html) you can do both in a single function:
 ```suggestion
@pytest.mark.django_db
@pytest.mark.parametrize('client_method', ['get_object', 'put_object'])
def test_s3_signed_url(api_client, client_method, user):
    """Test that a url string is given to the client."""
    api_client.login(username=user.username, password="test_password")
    response = api_client.post("/v1/reports/signed_url/", {
        "file_name": "test.txt",
        "file_type": "plain/text",
        "client_method": client_method
    })

    assert response.status_code == status.HTTP_200_OK
    assert response.data['signed_url']
```